cluster/gce/util.sh

```diff
@@ -742,6 +742,7 @@ function construct-linux-kubelet-flags {
 
   if [[ "${master}" == "true" ]]; then
     flags+=" ${MASTER_KUBELET_TEST_ARGS:-}"
+    flags+=" --register-with-taints=node-role.kubernetes.io/master=:NoSchedule"
```

Comment:
This should probably be in the if statement just below, since it doesn't make sense to add the taints unless we're using a registered kubelet for the control plane.

Reply:
Fixed